### DISCUSSION TITLE
Is it possible to define a featured image in the same directory as the note when it is NOT a page bundle?

### DISCUSSION THREAD

**Discussion by sosukeinu** [jpanther/congo#534] [Q&A] [answered]
I can't seem to figure out how to get an image to be found. Thank you

**Comment by jpanther** [comment, marked as answer]
This isn't currently possible as the featured image comes from the page resources and this requires a page bundle. You'd need to override the `single.html` template if you want to add custom logic to locate the image file somewhere else.

**Reply by sosukeinu** [reply]
Thank you.
